UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -75,7 +75,7 @@ bool Renderer::Init()
         return false;
     }
 
-    LOG_INFO("Renderer initialization started.");
+    spdlog::get(utility::Logger::LoggerName())->info("Renderer initialization started.");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.